src/Spout/Writer/Common/Entity/Row.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Entity;
+
+use Box\Spout\Writer\Common\Entity\Style\Style;
+use Box\Spout\Writer\Common\Manager\Style\StyleMerger;
+
+class Row
+{
+    /**
+     * The cells in this row
+     * @var array
+     */
+    protected $cells = [];
+
+    /**
+     * The row style
+     * @var null|Style
+     */
+    protected $style = null;
+
+    /**
+     * @var StyleMerger
+     */
+    protected $styleMerger;
+
+    /**
+     * Row constructor.
+     * @param array $cells
+     * @param Style|null $style
+     */
+    public function __construct(array $cells = [], Style $style = null)
```

Comment:
Then to be consistent with the naming of the other objects, you can actually rename that to RowBuilder and add a `build` function that returns the row with the added cells and correct style.

```
$row = new Row();
$row
    ->addCell(new Cell('G', $greenLight))
    ->setStyle($rowStyleItalicBold);
```
would become
```
$row = (new RowBuilder())
    ->addCell(new Cell('G', $greenLight))
    ->setStyle($rowStyleItalicBold);
    ->build()
```